plugins/blip/basics/logic/actions.js

```diff
@@ -72,7 +105,7 @@ function clearSelected(opts) {
   return opts;
 }
 
-function setSelected(opts, selectedKey) {
+basicsActions.setSelected = function(opts, selectedKey) {
```

Comment:
Seems fine on the surface, though it may intersect with my comment about moving more of the new code into the data munger? Not sure, feel free to ping me if you want to discuss.